jb-yoda/src/main/java/life/jukebox/JukeboxCreatedMessageReceiver.java

```diff
@@ -0,0 +1,18 @@
+package life.jukebox.yoda;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.SubscriptionName;
+
+public class JukeboxCreatedMessageReceiver implements MessageReceiver {
+
+  @Override
+  public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
+    System.out.println("Message Id: " + message.getMessageId());
+    System.out.println("Data: :" + message.getData().toStringUtf8());
```

Comment:
Let's get this info out on one line for easier digestion
System.out.printf("Message [%s]: '%s'\n", message.getMessageId(), gessage.getData().toStringUtf8())

Reply:
fixed